commit ffabd3937aea799d2779e8b1b0a5385eebadd452
Author: Andreas Schwab <schwab@suse.de>
Date:   Wed Aug 21 10:19:01 2013 +0200

    Fix dependencies for stdlib/tst-tls-atexit

2013-08-26  Andreas Schwab  <schwab@suse.de>

	* stdlib/Makefile (LDFLAGS-tst-tls-atexit): Remove.
	($(objpfx)tst-tls-atexit): Add dependencies here instead.

diff --git a/stdlib/Makefile b/stdlib/Makefile
index 17d80e0..b76c97a 100644
--- a/stdlib/Makefile
+++ b/stdlib/Makefile
@@ -161,6 +161,6 @@ $(objpfx)tst-strtod6: $(link-libm)
 
 tst-tls-atexit-lib.so-no-z-defs = yes
 
-LDFLAGS-tst-tls-atexit = $(common-objpfx)nptl/libpthread.so \
+$(objpfx)tst-tls-atexit: $(common-objpfx)nptl/libpthread.so \
 			 $(common-objpfx)dlfcn/libdl.so
 $(objpfx)tst-tls-atexit.out: $(objpfx)tst-tls-atexit-lib.so
